test/specs/elements/Container/Container-test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import Container from 'src/elements/Container/Container'
+import * as common from 'test/specs/commonTests'
+
+describe('Container', () => {
+  common.isConformant(Container)
+  common.rendersChildren(Container)
+  common.hasUIClassName(Container)
+
+  common.propKeyOnlyToClassName(Container, 'text')
+  common.propKeyAndValueToClassName(Container, 'aligned')
```

Comment:
This is the test that is failing.  This test will ensure that the component has the classNames:

`left aligned`
`right aligned`
`center aligned`
`justified aligned` <-- it is asserting this but it is wrong because we are overriding this value

I'm merging a new propUtil to handle the className logic.  I'm also merging a new common test to ensure the `aligned` prop is correctly implemented.  I think this makes sense since it has it's own className logic and it is used in many components.

Once merged, we'll be able to do this in `cx`:

```js
cx(
  useAlignedProp(aligned),
)
```

And this in tests:

```js
common.implementsAlignedProp(Container)
```